CHANGELOG.md

```diff
@@ -9,6 +9,96 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.9.0] - 2021-12-11
+
+## What’s Changed
+
+This release brings about several great new additions including:
+- dbt 1.0.0 compatibility.
+- CLI and Simple API paramaters to provide custom paths to config files.
+- Refinement to Simple API (**BREAKING CHANGE**).
```

Comment:
Probably good to explain what the breaking change is:

```suggestion
- Refinement to Simple API to return parse output on JSON rather than internal SQLFluff object (**BREAKING CHANGE**).
```

Reply:
Did you mean "to return parse output in JSON"?

Reply:
Yup. Updated.